pcdsdevices/lasers/rfof.py

```diff
@@ -11,19 +10,569 @@
+        ":RX_INT_P", kind="normal", doc="Internal pressure (mbar)"
+    )
+    int_temp_press_sens = Cpt(
+        EpicsSignalRO,
+        ":RX_INTP_T",
+        kind="normal",
+        doc="Internal temperature (pressure sensor) (C)",
+    )
+
+
+class ItechRfofTx(BaseInterface, Device):
+
+    """Itech RFoF transmitter PVs."""
+
     # Transmitter adds a single control PV
-    enable_laser = Cpt(EpicsSignal, ':ENABLELASER', kind='normal')
+    low_drift_opt_link_lock = Cpt(
+        EpicsSignalRO,
+        ":TX_PC1",
+        kind="normal",
+        doc="Low-drift optical link loop lock (uV)",
+    )
+    las_and_mod_loop_lock = Cpt(
+        EpicsSignalRO,
+        ":TX_PC2",
+        kind="normal",
+        doc="Laser and modulator loop lock (uV)",
+    )
+    pd1_current_return = Cpt(
+        EpicsSignalRO,
+        ":TX_PD1_I",
+        kind="normal",
+        doc="PD1 current - returned optical signal (uA)",
+    )
+    pd2_current_transmit = Cpt(
+        EpicsSignalRO,
+        ":TX_PD2_I",
+        kind="normal",
+        doc="PD2 current - transmitted \
+                               optical signal (uA)",
+    )
+    las_diode_sp = Cpt(
+        EpicsSignalRO,
+        ":TX_LD_S", kind="normal", doc="Laser diode setpoint (uA)"
+    )
+    las_temp = Cpt(
+        EpicsSignalRO,
+        ":TX_LAS_T", kind="normal", doc="Laser temperature (C)"
+    )
+    comp_spool_temp = Cpt(
+        EpicsSignalRO,
+        ":TX_SPL_T",
+        kind="normal",
+        doc="Compensation spool temperature (C)",
+    )
+    comp_spool_temp_sp = Cpt(
+        EpicsSignalRO,
+        ":TX_SPL_S",
+        kind="normal",
+        doc="Compensation spool temperature setpoint (C)",
+    )
+    input_rf_power = Cpt(
+        EpicsSignalRO,
+        ":TX_RFIN", kind="normal", doc="Input RF power (dBm)"
+    )
+    pd1_rf_power_return = Cpt(
+        EpicsSignalRO,
+        ":TX_PD1_RF",
+        kind="normal",
+        doc="PD1 RF power - returned \
+                              optical signal (dBm)",
+    )
+    pd2_rf_power_transmit = Cpt(
+        EpicsSignalRO,
+        ":TX_PD2_RF",
+        kind="normal",
+        doc="PD2 RF power - transmitted \
+                                optical signal (dBm)",
+    )
+    las_opt_power = Cpt(
+        EpicsSignalRO,
+        ":TX_LAS_P", kind="normal", doc="Laser optical power (mW)"
+    )
+    rf_output_mzm_power = Cpt(
+        EpicsSignalRO,
+        ":TX_RFO", kind="normal", doc="RF output power to MZM (dBm)"
+    )
+    las_current = Cpt(
+        EpicsSignalRO,
+        ":TX_LAS_I", kind="normal", doc="Laser current (mA)"
+    )
+    elect_phase_shift_v = Cpt(
+        EpicsSignalRO,
+        ":TX_PS_V",
+        kind="normal",
+        doc="Electrical phase shifter voltage (V)",
+    )
+    attenuator_setpoint_v = Cpt(
+        EpicsSignalRO,
+        ":TX_ATTS_V",
+        kind="normal",
+        doc="Variable RF attenuator \
+                                controller setpoint voltage (V)",
```

Comment:
I think the result of this is a string that's something like:
```
"Variable RF attenuator                                 controller setpoint voltage (V)"
```
Due to the way the line continuation is handled.

The simplest way to get a string to format correctly while using multiple lines is something like:
```suggestion
        doc=(
            "Variable RF attenuator "
            "controller setpoint voltage (V)"
        ),
```

Reply:
Several other doc= kwargs in the module here are continued similarly to this one

Reply:
Good to know!